server/activityItem/queries.ts

```diff
@@ -379,10 +386,10 @@ export const createPubReviewCommentAddedActivityItem = async (
 export const createPubReviewUpdatedActivityItem = async (
 	actorId: null | string,
 	reviewId: string,
-	oldReview: types.Review,
+	oldReview: ReviewNew,
 ) => {
-	const review: types.Review = await ReviewNew.findOne({ where: { id: reviewId } });
-	const pub: types.Pub = await Pub.findOne({ where: { id: review.pubId } });
+	const review = expect(await ReviewNew.findOne({ where: { id: reviewId } }));
+	const pub = expect(await Pub.findOne({ where: { id: expect(review.pubId) } }));
```

Comment:
Just to make sure this is reviewed at least once: I very often remove the hand typed return type like this, because it throws an error and is wrong, `findOne` does not return a serialized Model.

I could have also done `Review.findOne(...) as types.Review`. Because of the way the types are defined now this will not error, but it will still be incorrect.